jsonnetlib/config.jsonnet

```diff
@@ -8,6 +8,8 @@ assert std.length(v) > 0 : 'version is empty';
   defaultNamespace: 'default',
   port: 4000,
   baseHost: '',
+  cronCPURequest: '200m',
+  cronCPULimit: '1',
```

Comment:
Thanks for adding these.

Considering we are introducing the function with default values at the same time, it's good to set a larger value, we can decrease them after having them set in individual apps. But looking at what we have now with Deployment Pods, I would use smaller values here. How about `50m` and `500m`?

Reply:
@idealhack Done :)